gluten-core/src/main/scala/org/apache/gluten/expression/ExpressionMappings.scala

```diff
@@ -354,6 +354,5 @@ object ExpressionMappings {
       .toMap[Class[_], String]
   }
 
-  var expressionExtensionTransformer: ExpressionExtensionTrait =
-    DefaultExpressionExtensionTransformer()
+  var expressionExtensionTransformer: ExpressionExtensionTrait = _
```

Comment:
I'd suggest to use getter and setters to avoid directly accessing the variable.

In the setter, we can do a check to make sure it was not set.
Similarly in the getter, we can do a check to make sure it was set.

Reply:
@zhztheplayer Added the get and set method.